app/models/post.rb

```diff
@@ -1,14 +1,20 @@
 class Post < ApplicationRecord
+  belongs_to :user
   has_many :likes, foreign_key: :post_id
   has_many :comments, foreign_key: :post_id
-  belongs_to :author, class_name: 'User'
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
   after_save :update_author_post_counter
 
   def update_author_post_counter
-    author.increment!(:posts_counter)
+    user.increment!(:PostsCounter)
   end
 
+  private
+
   def recent_comment
```

Comment:
- [ ] As demanded by the project requirements, kindly add unit test for the `recent_comment` method. You are expected to add unit tests for **all of your models' methods and validations.**